### ISSUE TITLE
log file not found (error on startup)

### ISSUE THREAD

**Issue by timotheyca** [dmdhrumilmistry/SafePass#8]
how to replicate:
* clone repository
* install (`-e .`)
* run (`-m`)
what happens:
```
FileNotFoundError: [Errno 2] No such file or directory: '(some path)\\lib\\site-packages\\safepass\\logs\\SafePass.log'
```

**Comment by timotheyca** [comment]
`paths.py` should use `__path__` instead of sys.path to find package location

**Comment by timotheyca** [comment]
```python
root = __file__, '..'
log_file = join(*root,'logs','SafePass.log')
PASSWORD_DB = join(*root,'db','passwords.db')
USER_DB = join(*root,'db','users.db')
login_ui = join(*root,'ui','login.ui')
create_acc_ui = join(*root,'ui','CreateAccount.ui')
pass_table_ui = join(*root,'ui','PasswordsTable.ui')
save_info_ui = join(*root,'ui','SaveInfo.ui')
```

**Comment by dmdhrumilmistry** [comment]
Nice suggestion, I'll update the code. I've fixed various issues in the new release v1.0.3, now you can directly install SafePass using pip. 
So now there's no need to clone and install, pip will install safepass for you as a module.
```
pip install pysafepass
```
Link: [PyPi PySafePass](https://pypi.org/project/PySafePass/)
